azure-devops/projects/cgn-onboarding-portal-projects/hub-spid-login-ms.tf

```diff
@@ -0,0 +1,339 @@
+variable "hub-spid-login-ms" {
+  default = {
+    repository = {
+      organization   = "pagopa"
+      name           = "hub-spid-login-ms"
+      branch_name    = "master"
+      pipelines_path = ".devops"
+    }
+    pipeline = {
+      cache_version_id = "v1"
+      dev = {
+        deploy_type                               = "production_slot" #or staging_slot_and_swap
```

Comment:
So far we do not have a subscription for dev. We already have to many environments :-)

Reply:
ahahah